oauth2client/crypt.py

```diff
@@ -95,7 +97,107 @@ def make_signed_jwt(signer, payload):
     return b'.'.join(segments)
 
 
-def verify_signed_jwt_with_certs(jwt, certs, audience):
+def _verify_signature(message, signature, cert_list):
+    """Verifies signed content using a list of certificates.
+
+    Args:
+        message: string or bytes, The message to verify.
+        signature: string or bytes, The signature on the message.
+        cert_list: list, certificates in PEM format.
```

Comment:
@nathanielmanistaatgoogle FYI this is now `cert_list`.

As for the `AppIdentityError`, I suggested in the previous comment (now hidden as "outdated" by GitHub) that we punt on it and follow up after.